src/devtools/client/debugger/src/components/TestInfo/ContextMenu.tsx

```diff
@@ -113,9 +125,9 @@ function ContextMenu({
       </div>
       <div
         className={classnames("ContextMenuItem", {
-          disabled: testStep.absoluteStartTime === currentTime,
+          disabled: canJumpToBefore,
         })}
-        onClick={testStep.absoluteStartTime === currentTime ? undefined : onJumpToBefore}
+        onClick={canJumpToBefore ? undefined : onJumpToBefore}
```

Comment:
While we're at it, we should probably remove jumping before/after in an assert. Since the way we've been framing it, assert is a single point in time.